site/src/examples/tokenized-input-next/Disabled.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactElement } from "react";
+import { TokenizedInputNext } from "@salt-ds/lab";
+
+export const Disabled = (): ReactElement => (
+  <TokenizedInputNext
+    disabled
+    initialSelectedItems={["Value 1", "Value 2", "Value 3"]}
```

Comment:
The wording of these props need to be updated, we stopped using the `initial` wording and have used `default` everywhere in our code